### ISSUE TITLE
Unable to export and progress bar is displayed all the time

### ISSUE THREAD

**Issue by rakeshnambiar** [gitlab-ce-export-to-csv/gitlab-ce-export-to-csv.github.io#4]
Unable to export and progress bar is displayed all the time. I waited for an hour and didn't get any response. Latest on I tried a small scope with a specific milestone and status is open. But still the behaviour is same. Also, noticed the `Group` dropdown is empty for me. is this somehow connected to the user permission? I am only the developer access in this particular gitlab repo and it CE. Not the EE.

**Comment by rakeshnambiar** [comment]
@natanielpaiva pls advise

**Comment by rakeshnambiar** [comment]
I also checked this along with the owner of this rspo and he's also getting the error `Problem! Verify your credentials 401 Unauthorized.`

**Comment by natanielpaiva** [comment]
Can be the token.
You need use the options for create token:
https://drive.google.com/file/d/1CZ7ECv4vcapM0vZVMgutxigMvmO8D6YZ/view?usp=sharing

**Comment by rakeshnambiar** [comment]
Yes, I selected the API option. Hope the expiry date is not mandatory?

**Comment by natanielpaiva** [comment]
Yes, is mandatory. Put on at least one more day

**Comment by rakeshnambiar** [comment]
I selected almost a week time, but stil lthe behavious is same. When I tab out after pasting the token the same error is displayed.

**Comment by natanielpaiva** [comment]
Is gitlab.com? Or your server?

**Comment by rakeshnambiar** [comment]
https://gitlab.coko.foundation

**Comment by natanielpaiva** [comment]
Can be proxy of your server's network

**Comment by rakeshnambiar** [comment]
I don't know. It's managed by a different company.

**Comment by rakeshnambiar** [comment]
What sort of proxy issue you are suspecting?
